src/component/todo-list-app.ts

```diff
@@ -0,0 +1,61 @@
+import '../component/todo-input/todo-input';
+import '../component/todo-list/todo-list';
+import '../component/todo-info/todo-info';
+
+import { filterStore } from '../utils/filter-store';
+import { FilterType } from '../models/filter';
+import { DndUtil } from '../utils/dnd-util/dnd-util';
+export class TodoListApp extends HTMLElement {
+  private elementContainer: HTMLElement;
+  private dnd: DndUtil | null = null;
+  private useDnd: boolean = false;
+  constructor(elementContainer: HTMLElement, useDnd?: boolean) {
+    super();
+    this.elementContainer = elementContainer;
+
+    if (useDnd) {
+      this.useDnd = true;
+      window.document.body.addEventListener('list-render', this.addDnd);
+    } else {
+      this.useDnd = false;
+      window.document.body.removeEventListener('list-render', this.addDnd);
+    }
```

Comment:
```suggestion
    this.useDnd = !!useDnd;
```
이벤트 리스너 해제 조건이 잘못되었습니다.
dnd기능을 사용안할때 이벤트 리스너를 해제하는것보단,
dnd기능을 활성화하면 이벤트 등록해주고 removeEventListener는 인스턴스가 파괴될때 해제해주는게 좋겠습니다.